InternetHospitalWebUI/src/app/Components/doctor-page/doctor-page.component.html

```diff
@@ -0,0 +1,33 @@
+<div [fxLayout]="showLikeRow ? 'row' : 'column'">
+  <mat-card fxFlex="40%" class="mat-elevation-z8 main-info">
+    <p class="doctor-name">{{doctor?.firstName}} {{doctor?.secondName}} {{doctor?.thirdName}}</p>
+    <div class="doctor-photo-container">
+      <img [src]="doctor?.avatarURL">
+    </div>
+    <div class="doctor-info">
+      <p><span class="info-name">Specialization:</span><span class="info-value">{{doctor?.specialization}}</span></p>
+      <p><span class="info-name">Birth date:</span><span class="info-value">{{doctor?.birthDate}}</span></p>
+      <p><span class="info-name">Address:</span><span class="info-value">{{doctor?.address}}</span></p>
+      <p><span class="info-name">Phone:</span><span class="info-value">{{doctor?.phone}}</span></p>
+    </div>
+    <div class="doctor-rating">
+      <span class="marks negative">
+        <mat-icon>thumb_down</mat-icon> 1000
+      </span>
+      <span class="marks positive">
+        <mat-icon>thumb_up</mat-icon> 1000
+      </span>
+    </div>
+  </mat-card>
+  <div fxFlex="60%" style="margin-top: 10px; margin-left: 10px; vertical-align: top;">PLACE FOR APPOINTMENT</div>
```

Comment:
Remove style to .sccs file

Reply:
It's just mock objects, I put styles in them just because it would be easier to delete them in future.